pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.596.1</version>
```

Comment:
I don't know that "previous attempts" or "Xstream issues they encounteted".

> should plugins not move forward with the code base?

They should not unless the developer know what he / she do and can explain why he / she do that.